apps/hospitals/models.py

```diff
@@ -35,6 +35,13 @@ class Hospital(models.Model):
     email = models.EmailField("E-mail", blank=False, max_length=254)
 
 
+class HospitalUser(models.Model):
+    username = models.CharField("Usuário", blank=False, max_lengt=254)
+    password = models.CharField("Senha", blank=False, max_lengt=254)
```

Comment:
Creio que **max_lengt=254** seja muito.
```suggestion
    username = models.CharField("Usuário", blank=False, max_lengt=150)
    password = models.CharField("Senha", blank=False, max_lengt=100)
```